### PR TITLE
Update

### PR DESCRIPTION
**Pull request by ptbrowne**
Primary goal was to be able to use SWC, so had to remove babel, so had to remove lingui and used i18next.

- chore: Install and configure i18next, next-i18next
- feat: Add codemod
- refactor: Run codemod
- chore: Convert messages.po to json
- feat: Use next-i18next
- fix: Fixing translation bugs
- feat: Remove babel, upgrade to next 12
- fix: Small fixes + prettier on json

### CODE REVIEW

File: scripts/codemod-i18next.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+// jscodeshift can take a parser, like "babel", "babylon", "flow", "ts", or "tsx"
+// Read more: https://github.com/facebook/jscodeshift#parser
+export const parser = "tsx";
+
+export default function transformer(file, api) {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+  root
```
Comment (bprusinowski) [inline scripts/codemod-i18next.js:8]: wow, awesome!

**Review by bprusinowski — COMMENTED** [review]
Code looks great! 👏 

But when going to a preview page, it seems that some translations are not detected properly, see below:

Screenshot 2023-02-20 at 14 02 57

Also it seems that the "List filtern" is not updated when changing language. Generally I think the right sidebar's translations are incorrect 😱 

It seems that the page title also missed translation ⬇️ 

Screenshot 2023-02-20 at 14 03 47

Also not sure if details page are translated 👀

**Comment by ptbrowne** [comment]
Thanks, the problem where appearing on Vercel and fixed through https://github.com/visualize-admin/electricity-prices-switzerland/pull/66/commits/c5bbe5d3d3b7c9ee8dd90ef4a1090bd7cd215c43. I also removed completely lingui. Thanks for noticing Bartosz, I had seen the failed preview on Friday and forgotten about it 🙄 !

File: src/next-i18next.config.js [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
 module.exports = {
   i18n: {
     defaultLocale: "de",
     locales: ["de", "fr", "it"],
   },
+  /**
+   * Below line necessary for Vercel
+   * @see https://github.com/i18next/next-i18next/issues/462
+   */
+  localePath: path.resolve("./public/locales"),
```
Comment (bprusinowski) [inline src/next-i18next.config.js:12]: 👍

**Pull request merged**